src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,137 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.utility.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createQuery = "INSERT INTO manufacturers(name,country) VALUES(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufactureStatement =
+                        connection.prepareStatement(createQuery,
+                                                    Statement.RETURN_GENERATED_KEYS)) {
+            setStatementParameters(manufacturer, createManufactureStatement);
+            createManufactureStatement.executeUpdate();
+            setIdToManufacturesObj(manufacturer, createManufactureStatement);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not create manufacturer " + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String selectQuery = "SELECT *"
+                + " FROM manufacturers"
+                + " WHERE is_deleted='FALSE' AND id=?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufactureStatement =
+                        connection.prepareStatement(selectQuery)) {
+            getManufactureStatement.setLong(1, id);
+            boolean successes = getManufactureStatement.execute();
+            if (!successes) {
+                return Optional.empty();
+            }
+            ResultSet resultSet = getManufactureStatement.getResultSet();
+            while (resultSet.next()) {
+                Manufacturer manufacturer = new Manufacturer();
+                enrichManufacturerObj(manufacturer, resultSet);
+                return Optional.of(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not get manufacturer by id " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        String selectAllQuery = "SELECT *"
+                + " FROM manufacturers"
+                + " WHERE is_deleted='FALSE'";
```

Comment:
```suggestion
                + " WHERE is_deleted=FALSE";
```
we don't need to escape false

Reply:
This is because you used varchar and you need tinyint, please fix it